aptos-move/aptos-release-builder/src/components/feature_flags.rs

```diff
@@ -76,6 +76,7 @@ pub enum FeatureFlag {
     GasPayerEnabled,
     AptosUniqueIdentifiers,
     BulletproofsNatives,
+    BulletproofsBatchNatives,
```

Comment:
this has to go the end of the list so existing feature -> id mapping remain the same.

Reply:
oh maybe for this enum it doesn't matter... still recommend put it at the end though